packages/bpk-component-text/src/BpkText.tsx

```diff
@@ -58,6 +58,21 @@ export const TEXT_STYLES = {
   editorial3: 'editorial-3',
 } as const;
 
+export const TEXT_COLORS = {
+  textDisabled: 'text-disabled',
+  textDisabledOnDark: 'text-disabled-on-dark',
+  textError: 'text-error',
+  textHero: 'text-hero',
+  textLink: 'text-link',
+  textOnDark: 'text-on-dark',
+  textOnLight: 'text-on-light',
+  textPrimary: 'text-primary',
+  textPrimaryInverse: 'text-primary-inverse',
+  textSecondary: 'text-secondary',
+  textSuccess: 'text-success',
+} as const;
```

Comment:
Product Colors [currently documented](https://www.skyscanner.design/latest/foundations/colours/overview-DOFKsdZ0#section-product-colours--5b) for text 

<img width="600" height="756" alt="image" src="https://github.com/user-attachments/assets/60a9f28d-9d41-41d8-a4e4-164b99d0b910" />